lib/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ add_subdirectory(administration)
 add_subdirectory(hasher)
 add_subdirectory(rbudp)
 add_subdirectory(metadata)
+#add_subdirectory(genquery1)
```

Comment:
Don't forget to handle this. Requires discussion first.

Reply:
Restored this.

I removed the circular dependency between rcMisc.cpp and the parser, therefore resolving the issue.